docs/windows.md

```diff
@@ -0,0 +1,60 @@
+# Experimental Windows support
```

Comment:
"Windows containers"

Reply:
Sure, fixing that, plus in the description.